src/Tribe/I18n.php

```diff
@@ -150,14 +194,30 @@ protected function with_locale( $locale, callable $do, array $args = [] ) {
 	/**
 	 * Compiles the translations for a set of strings iterating on a set of domains.
 	 *
+	 * The 4th argument is a bitmask to control the compiled translations.
+	 * E.g. `$i18n->compile_translations( $strings, $domains, I18n::COMPILE_STRTOLOWER);` will only compile
+	 * translations of the strings in their `strtolower` versions.
+	 * Combine the flags using the usual PHP syntax: `I18n::COMPILE_INPUT | I18n::COMPILE_STRTOLOWER` to compile
+	 * only the translation of the string as input and in their lowercase version.
+	 *
 	 * @since 5.1.1
+	 * @since TBD   Add support for the $flags argument.
 	 *
 	 * @param array<string,array|string> $strings The set of strings to compile the translations for.
 	 * @param string|array<string>       $domains The domain(s) that should be used to compile the string translations.
+     * @param int                        $flags   An integer resulting from the combination of compilation flags;
```

Comment:
:no_entry_sign: Tabs must be used to indent lines; spaces are not allowed

<sub>hash: 00cdfc0e9fc833ec79192276e1e41a6b</sub>